### PR TITLE
Security improvements

### PR DESCRIPTION
**Pull request by imollov**

### CODE REVIEW

File: contracts/DssVestTopUp.sol [hunk 1]
```diff
@@ -68,6 +79,17 @@ contract DssVestTopUp is IUpkeepRefunder, Ownable {
         uint256 _maxDepositAmt,
         uint256 _threshold
     ) {
+        require(_dssVest != address(0), "invalid dssVest address");
+        require(_daiJoin != address(0), "invalid daiJoin address");
+        require(_vow != address(0), "invalid vow address");
+        require(_paymentToken != address(0), "invalid paymentToken address");
+        require(_keeperRegistry != address(0), "invalid keeperRegistry address");
+        require(_swapRouter != address(0), "invalid swapRouter address");
+        require(_linkToken != address(0), "invalid linkToken address");
```
Comment (mradkov) [inline contracts/DssVestTopUp.sol:88]: Wouldn't it be better to generalize the require for non 0x0 address? Or do you want to keep the error messages more explicit?

File: contracts/DssVestTopUp.sol [hunk 2]
```diff
@@ -68,6 +79,17 @@ contract DssVestTopUp is IUpkeepRefunder, Ownable {
...
+        require(_minWithdrawAmt > 0, "invalid minWithdrawAmt");
+        require(_maxDepositAmt > 0, "invalid maxDepositAmt");
+        require(_threshold > 0, "invalid threshold");
+
```
Comment (mradkov) [inline contracts/DssVestTopUp.sol:92]: Same as above. Can we generalize check for `>0` ?

File: contracts/DssVestTopUp.sol (hunk 1 above)
Comment (imollov) [reply contracts/DssVestTopUp.sol:88]: Not sure how to do it in a way that will actually simplify it without obscuring it for the sake of brevity. Open for suggestions.

**Review by mradkov — APPROVED** [review]
LGTM

**Pull request merged**